### PR TITLE
Added docstring documentation to all functions

### LINKED ISSUE #27: Add docstring documentation to all functions
Add documentation to functions using docstring and common Python documentation conventions.

### PR DESCRIPTION
**Pull request by gabb222**
Fixes #27

### CODE REVIEW

**Comment by gabb222** [comment]
@Theia-Scientific/developers ready for review

File: tspng/extraction.py [hunk 1]
```diff
@@ -15,6 +25,16 @@ def extract_from_file(path, mime_type=MIME_TYPE):
     return dict
 
 def extract_from_files(paths, mime_type=MIME_TYPE):
+    '''
+    Returns a nested dictionary of metadata from a list of TSPNG file paths.
+
+        Parameters:
+                path (list[str]): List of file paths
+                mime_type (str): Optional; Media type of file
```
Comment (volks73) [inline tspng/extraction.py:33]: See other comment about adding the default value since it is optional.

File: tspng/test_extraction.py [hunk 2]
```diff
@@ -2,14 +2,17 @@
 
 from tspng.extraction import extract_from_file,extract_from_files
 
-def test_hello_world():
-    print('Hello World!')
-
 def test_extract_from_file():
+    '''
```
Comment (volks73) [inline tspng/test_extraction.py:6]: This is great, but docstrings on tests are not required. We will not convert the comments from tests into a help manual or hosted documentation.

File: tspng/extraction.py [hunk 3]
```diff
@@ -6,6 +6,16 @@
 import os
 
 def extract_from_file(path, mime_type=MIME_TYPE):
+    '''
+    Returns the metadata from a TSPNG file as a dictionary.
+
+        Parameters:
+                path (str): Path to a file as a string
+                mime_type (str): Optional; Media type of file
```
Comment (volks73) [inline tspng/extraction.py:14]: I would add something about it defaulting to the `MIME_TYPE` constant and the value being `application/vnd.theiascope.io+json`

**Review by volks73 — APPROVED** [review]
Looks good.

This issue and PR are about getting the documentation started. I have some small, non-blocking suggestions.

File: tspng/test_extraction.py (hunk 2 above)
Comment (gabb222) [reply tspng/test_extraction.py:6]: I will keep them in for now

File: tspng/extraction.py (hunk 3 above)
Comment (gabb222) [reply tspng/extraction.py]: Added something for now

**Pull request merged**